user-management.yaml

```
@@ -654,6 +675,13 @@ paths:
         - uc4_auth: [] 
       description: "Returns all admins, may only be used by administrators"
```

is this still true, or can all fetch users using a list of usernames, similar to above?
```suggestion
      description: ""Without parameter \"usernames\", returns all admins, and may only be used by administrators. With \"usernames\" set, returns all admins with the given usernames, which can be invoked by all roles.""
```